velox/substrait/VeloxToSubstraitPlan.h

```diff
@@ -48,6 +48,12 @@ class VeloxToSubstraitPlanConvertor {
       const core::PlanNodePtr& planNode,
       ::substrait::Rel* rel);
 
+  /// Convert Velox PlanNode into Substrait RootRel.
+  void toSubstrait(
+      google::protobuf::Arena& arena,
+      const core::PlanNodePtr& planNode,
+      ::substrait::RelRoot* rootRel);
```

Comment:
What does RootRel represent?

Reply:
The root of a relation tree. Please see https://github.com/substrait-io/substrait/blob/main/proto/substrait/plan.proto#L15.
In json file, it will looks like:
![image](https://user-images.githubusercontent.com/52772496/176576741-a2f1699a-0470-4462-9a16-f03416ab179f.png)